### DISCUSSION TITLE
Unable to create, update, insert in table.

### DISCUSSION THREAD

**Discussion by nefagi-01** [trinodb/trino-python-client#244] [Q&A] [answered]
Hi everyone,

I am trying to use trino-python-client to create, update and insert into a table.
Even if I am able to do this from the basic Trino command line, this is not possible with trino-python-client.
Indeed with trino-python-client I am only able to select from the table, whereas all the queries containing insert, update or create are not executed.
How can I solve this??

**Comment by ebyhr** [comment, marked as answer]
Did you put `fetchall()` after `execute()` method? e.g.
```python
    cur.execute('INSERT INTO sometable VALUES (4, 5, 6)')
    cur.fetchall()
```

**Reply by nefagi-01** [reply]
I didn't. Now I tried with `cur.fetchall()` and it works, thank you! 👍
